### PR TITLE
[Maintenance] Update JS dependencies

### PR DESCRIPTION
**Pull request by jakubtobiasz**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.12                   |
| Bug fix?        | no                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no  |
| Related tickets | -                      |
| License         | MIT                                                          |

### CODE REVIEW

File: bin/fix-semantic-css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (coldic3) [inline bin/fix-semantic-css]: https://www.npmjs.com/package/@semantic-ui-react/css-patch WDYT?

**Review by coldic3 — APPROVED** [review]

File: bin/fix-semantic-css (hunk 1 above)
Comment (coldic3) [reply bin/fix-semantic-css]: In addition, this bash script doesn't cover all cases like BSD OS 😄

Comment (jakubtobiasz) [reply bin/fix-semantic-css]: Thanks for the package, I've committed replacement of my custom script with this one. It seems to work locally, let's wait for CI.

**Review by Zales0123 — APPROVED** [review]
Is there any info we should put into the UPGRADE file to make an upgrade easier? Nevertheless, seems good 👍

**Comment by jakubtobiasz** [comment]
@Zales0123 In terms of this PR no actions are required on the end user (developer) side, so there's nothing to add to the UPGRADE file.

**Comment by Zales0123** [comment]
Thank you, @jakubtobiasz! 🎉

**Pull request merged**